fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -78,6 +78,32 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
     if not ctx.invoked_subcommand:
         click.echo(cli.get_help(ctx))
 
+    if (
+        ctx.obj["CONFIG"].user.analytics_opt_out is None
+        and ctx.invoked_subcommand.lower() != "init"
+    ):
+        ctx.obj["CONFIG"].user.analytics_opt_out = bool(
+            input(OPT_OUT_COPY).lower() == "n"
+        )
+        ctx.obj["CONFIG"].cli.analytics_id = ctx.obj[
+            "CONFIG"
+        ].cli.analytics_id or generate_client_id(FIDESCTL_API)
```

Comment:
I think this may be unnecessary with the model setting it by default. Also, did we intend to remove the api `analytics_id` from being set here as well for the initial release with the long-term view of storing it in the database?